com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -117,17 +120,36 @@ internal static void DontSaveToLayout(EditorWindow wnd)
 
         void OnEnable()
         {
-            m_CoreConvertersList = new List<RenderPipelineConverter>();
-
             // This is the drop down choices.
             m_ConverterContainers = TypeCache.GetTypesDerivedFrom<RenderPipelineConverterContainer>();
+            foreach (var cont in m_ConverterContainers)
+            {
+                var container = (RenderPipelineConverterContainer)Activator.CreateInstance(cont);
+                m_Containers.Add(container);
+                m_ContainerChoices.Add(container.name);
+            }
+
+            if (m_ConverterContainers.Any())
+            {
+                GetConverters();
+            }
+        }
+
+        void GetConverters()
+        {
+            m_CoreConvertersList.Clear();
+            m_ConverterStates.Clear();
+            m_ItemsToConvert.Clear();
             var converterList = TypeCache.GetTypesDerivedFrom<RenderPipelineConverter>();
 
             for (int i = 0; i < converterList.Count; ++i)
             {
-                // Iterate over the converters
+                // Iterate over the converters that are used by the current container
                 RenderPipelineConverter conv = (RenderPipelineConverter)Activator.CreateInstance(converterList[i]);
-                m_CoreConvertersList.Add(conv);
+                if (conv.container == m_ConverterContainers[m_ContainerChoiceIndex])
```

Comment:
Were keeping duplicate information atm if I am not mistaken. We are storing both m_CurrentContainer and m_ConverterContainers[m_ContainerChoiceIndex]. Perhaps we could just use one of the two? Maybe m_CurrentContainer.GetType() would work here.

Reply:
Unconfused this!